### PR TITLE
Monkeypatch Win32::Registry#write

### PR DESCRIPTION
**Pull request by mcquin**
Resolves encoding errors

### CODE REVIEW

**Comment by chef-supermarket** [comment]
Hi. I am an automated pull request bot named Curry. There are commits in this pull request whose authors are not yet authorized to contribute to Chef Software, Inc. projects or are using a non-GitHub verified email address. To become authorized to contribute, you will need to sign the Contributor License Agreement (CLA) as an individual or on behalf of your company. [You can read more on Chef's blog.](https://www.getchef.com/blog/2014/06/23/changes-to-the-contributor-license-agreement-process)

## Non-GitHub Verified Committers

There are 1 commit author(s) whose commits are authored by a non-GitHub verified email address. Chef will have to manually verify that they are authorized to contribute.

[Please sign the CLA here.](https://supermarket.chef.io/become-a-contributor)

**Comment by chefsalim** [comment]
Looks good to me :+1:

**Comment by smurawski** [comment]
:+1:

**Comment by chef-supermarket** [comment]
Hi. Your friendly Curry bot here. Just letting you know that all commit authors have become authorized to contribute. I have added the "Signed CLA" label to this issue so it can easily be found in the future.

File: lib/chef/win32/unicode.rb [hunk 1]
```diff
@@ -53,3 +53,30 @@ def to_wstring
     Chef::ReservedNames::Win32::Unicode.utf8_to_wide(self)
   end
 end
+
+# https://bugs.ruby-lang.org/issues/11439
+if RUBY_VERSION =~ /^2\.1/
```
Comment (jaym) [inline lib/chef/win32/unicode.rb]: does this bug only affect ruby 2.1?
We still ship 2.0 in chef client

Comment (mcquin) [inline lib/chef/win32/unicode.rb]: Yes, I've tested it on 2.0. It's not an issue.

Comment (jaym) [inline lib/chef/win32/unicode.rb]: :+1:

**Pull request merged**